docs/developer-guide/printing-module.md

```diff
@@ -0,0 +1,114 @@
+# Printing Module
+
+The **printing module** of MapStore is a back-end service **not included  by default in the release package** that allows to create a printable PDF from the current map.
+
+!!! note
+    The **printing module** is required by the **Print plugin** of MapStore, so if you want to have the Print plugin working in you application, you have to include also the printing module in your MapStore installation.
+
+## Including the printing module in MapStore
+
+Because MapStore doesn't include the printing module by default, to use it you need to build rom the source a `MapStore.war` that includes it or add the missing files to an existing `MapStore.war`
+
+### Building from the Source
+
+If you want to include the printing module in your MapStore, building the source code you have to add the profile `printing` (profiles are the 2nd argument of the `build.sh` script, after the version):
```

Comment:
```suggestion
If you want to include the printing module in your MapStore, by building the source code, you have to add the profile `printing` (profiles are the 2nd argument of the `build.sh` script, after the version):
```

when you say "profiles are the 2nd" you mean you can add more profiles as second argument? if that is not the case i would change to "is the 2nd"

Reply:
Yes you can add more than one profile using `,` as separator.
We have only one other profile,  `release` (that creates the binary), so you can create a binary with the printing using `printing,release`

Reply:
ok, can you consider to add this detail  ?